src/emails/templates/HeaderStyles.tsx

```diff
@@ -40,6 +38,11 @@ export const HeaderStyles = () => {
         .hero_background {
           background: none !important;
         }
+
+        * {
+          background: #1e1e1e !important;
+          color: #ffffff !important;
+        }
```

Comment:
I'm not quite sure I follow what the intention is here, but isn't this a bit too wide of a selector? For example, buttons would no longer have a background colour anymore either:

![image](https://github.com/user-attachments/assets/198a573b-fab6-490e-8320-d07e66a1df76)

Reply:
Oh sorry, just noticed this PR wasn't ready for review yet, never mind!